common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerMsalController.java

```diff
@@ -144,25 +144,54 @@ public class BrokerMsalController extends BaseController {
 
     private BrokerOperationExecutor mOperationExecutor;
 
-    @VisibleForTesting
+    private String mMaxMsalBrokerProtocolVersion;
+
+    public BrokerMsalController(@NonNull final Context applicationContext,
```

Comment:
How do we ensure that OneAuth doesn't use this constructor and uses the other one?

Reply:
I'm working with nick on this as part of the feature pre-mortem.